lib/modules/platform/azure/index.ts

```diff
@@ -182,7 +184,14 @@ export async function getJsonFile(
   branchOrTag?: string
 ): Promise<any> {
   const raw = await getRawFile(fileName, repoName, branchOrTag);
-  return raw ? JSON5.parse(raw) : null;
+  const fileType = upath.extname(fileName);
+  if (!raw) {
+    return null;
+  }
+
+  return fileType === '.json5'
+    ? JSON5.parse(raw)
+    : parseJsonWithFallback(raw, fileName);
```

Comment:
we should extract a platform util function and call from all platforms

Reply:
Its a normal util function that parses json, and shouldn't differ per platform ... so how will that be beneficial?

Reply:
I think the point is that this is not unique to azure so should be a shared util

Reply:
yes, this conditional code is called from all platforms, so that's why I suggest to extract a util function

Reply:
Got it, i gues the parseJson you suggested below is related to this. I was seeing them separately.